backend/main.py

```
@@ -60,15 +95,23 @@ async def register(user: UserCreate, db: Session = Depends(get_db)):
         password=hashed_password,
         full_name=user.full_name,
         address=user.address,
-        phone=user.phone
+        phone=user.phone,
+        is_verified=False  
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)